tests/hipscat_import/catalog/test_resume_plan.py

```diff
@@ -141,3 +162,57 @@ def test_read_write_splitting_keys(tmp_path, small_sky_single_file, formats_head
     plan.gather_plan()
     split_keys = plan.split_keys
     assert len(split_keys) == 2
+
+
+@pytest.mark.dask
+def test_some_split_task_failures(tmp_path, dask_client):
+    """Test that we only consider split stage successful if all done files are written"""
+    plan = ResumePlan(tmp_path=tmp_path, progress_bar=False, input_paths=["foo1"])
+
+    ## Method doesn't FAIL, but it doesn't write out the done file either.
+    futures = [dask_client.submit(error_on_even, 1)]
+    with pytest.raises(RuntimeError, match="split stages"):
```

Comment:
I'm confused by this test and the lines below. Why would `error_on_even` raise a `RuntimeError` in this case? It is only taking single odd parameter, which it should be happy with.

Reply:
Yep. The behavior of erroring on even inputs is unused (used in the `test_pipeline_resume_plan.py` and I copy/pasted). Changed to use a simpler method that doesn't do ANYTHING.